addons/pos_restaurant/static/src/overrides/components/navbar/navbar.js

```diff
@@ -90,4 +99,40 @@ patch(Navbar.prototype, {
     get showEditPlanButton() {
         return true;
     },
+    async switchTable() {
+        const table_number = await makeAwaitable(this.dialog, NumberPopup, {
```

Comment:
Just to be sure, this means that we can't access a floating order with an alphabetical name? 
And if the floating order has the same number as a table, we give priority to the table.

Reply:
Yep and this could be changed later if we decide to change this but for the moment it is like that